rasa/utils/tensorflow/model_data_utils.py

```diff
@@ -41,6 +42,8 @@ def featurize_training_examples(
     Args:
         training_examples: the list of training examples
         attributes: the attributes to consider
+        type: feature type to consider; if set to None all types
+          (i.e. sequence and sentence) will be considered

```

Comment:
Could you please add the description at the end? Helps keep the order of arguments in the function signature same as in the docstring.

Reply:
done